server/command.go

```diff
@@ -216,6 +216,10 @@ func (p *Plugin) runListCommand(args []string, extra *model.CommandArgs) (bool,
 func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, error) {
 	issue, foreignID, err := p.listManager.PopIssue(extra.UserId)
 	if err != nil {
+		if err.Error() == "cannot find issue" {
```

Comment:
The idiomatic way would be to define an error type e.g. `ItemNotFound` and check for it here using `error.Is`.